src/typescript/typescript.ts

```diff
@@ -574,6 +571,8 @@ export class TypeScriptProject extends NodeProject {
 
     if (!this.parent && options.projenrcTs) {
       new ProjenrcTs(this, options.projenrcTsOptions);
+    } else {
+      this.tsconfigDev.addInclude(PROJEN_RC); // projenrc.js created in NodeProject needs to be added in tsconfigDev
```

Comment:
Not required for this PR, but if you are down for it.

```suggestion
    // when this is a root
    if (!this.parent) {
      if (options.projenrcTs) {
        new ProjenrcTs(this, options.projenrcTsOptions);
      } else {
        // projenrc.js created in NodeProject needs to be added in tsconfigDev
        const projenrcJs = Projenrc.of(this);
        if (projenrcJs) {
          this.tsconfigDev.addInclude(projenrcJs.filePath);
        }
      }
```

Reply:
I'll track down the rest.